contents/blog/best-mobile-app-ab-testing-tools.mdx

```diff
@@ -0,0 +1,317 @@
+
+## 3. Optimizely
+
+- **Free tier:** <span className="text-red text-lg">✖</span>
+- **Open source:** <span className="text-red text-lg">✖</span>
+- **Transparent pricing:** <span className="text-red text-lg">✖</span>
+- **Includes product analytics:** <span className="text-red text-lg">✖</span>
+- **Get started without talking to sales:** <span className="text-red text-lg">✖</span>
+- **Best for:** Marketing teams
+
+![Optimizely mobile app A/B tests](../images/blog/best-mobile-app-ab-testing-tools/optimizely-testing.png)
+
+### What is Optimizely?
+
+[Optimizely](/blog/posthog-vs-optimizely) is an all-in-one system for marketing that includes A/B testing **and** more – like content management, campaign planning, asset management, and checkout customizations. 
+
+It's built for marketers in large enterprises looking to optimize their content, apps, and e-commerce experiences.
+
+### Which platforms does Optimizely support?
+
+- **iOS:** <span className="text-green text-lg">✔</span>
+- **Android:** <span className="text-green text-lg">✔</span>
+- **Flutter:** <span className="text-green text-lg">✔</span>
+- **React Native:** <span className="text-green text-lg">✔</span>
+- **Web:** <span className="text-green text-lg">✔</span>
+- **Server:** <span className="text-green text-lg">✔</span>
+
+### How much does Optimizely cost?
+
+Optimizely's pricing is sales-driven and completely opaque, so it's not clear how much it costs. However, according to [Splitbase](https://splitbase.com/blog/optimizely-pricing), Optimizely will cost you at least $36,000 per year.
+
+## 4. Firebase
+
+- **Free tier:** <span className="text-green text-lg">✔</span>
+- **Open source:** <span className="text-red text-lg">✖</span> 
+- **Transparent pricing:** <span className="text-green text-lg">✔</span>
+- **Includes product analytics:** <span className="text-red text-lg">✖</span>
+- **Get started without talking to sales:** <span className="text-green text-lg">✔</span>
+- **Best for:** Apps already using Firebase's other features
+
+![Firebase mobile app A/B tests](../images/blog/best-mobile-app-ab-testing-tools/firebase.png)
+
+### What is Firebase A/B testing?
+
+[Firebase A/B testing](https://firebase.google.com/products/ab-testing) is built specifically for mobile apps and is completely free. It includes an easy to use interface for running experiments on Firebase's other app features, such as [cloud messaging](https://firebase.google.com/products/cloud-messaging) and [in-app messaging](https://firebase.google.com/products/in-app-messaging). 
+
+It's important to note that using A/B testing on Firebase requires installing [Google Analytics](/blog/posthog-vs-ga4) in your app. It's also not possible to run A/B tests on your web app or server.
+
+### Which platforms does Firebase support?
+
+- **iOS:** <span className="text-green text-lg">✔</span>
+- **Android:** <span className="text-green text-lg">✔</span>
+- **Flutter:** <span className="text-red text-lg">✖</span>
```

Comment:
Seems like Flutter is support? https://firebase.google.com/docs/flutter/setup?platform=ios